packages/vue/src/components/molecules/SfComponentSelect/SfComponentSelect.vue

```diff
@@ -20,7 +20,7 @@
     @keyup.down="move(1)"
     @keyup.enter="enter($event)"
   >
-    <div style="position: relative;">
+    <div style="position: relative">
```

Comment:
Do you have a different linter configuration? I think you were not editing these components